README.md

```diff
@@ -74,15 +74,16 @@ name: default
 
```

Comment:
readme: pluginSettings table: deployment_file_repository_path not always optional comment